plugins/flutter_aepcore/ios/Classes/FlutterAEPCorePlugin.m

```diff
@@ -112,7 +112,20 @@ - (void)handleDispatchEvent:(FlutterMethodCall *)call
 
 - (void)handleDispatchEventWithResponseCallback:(FlutterMethodCall *)call
                                             result:(FlutterResult)result {
-    NSDictionary *eventDict = (NSDictionary *)call.arguments;
+    NSDictionary *eventDict = (NSDictionary *)call.arguments[@"eventData"];
+    NSNumber *timeoutNumber = call.arguments[@"timeout"];
+    double timeout = 1;
```

Comment:
The default value of 1 is confusing. Can you change the logic to error out if it does not respond to selector and initialize the variable with the timeout value?